examples/flutter_gallery/lib/demo/pesto_demo.dart

```diff
@@ -123,17 +127,38 @@ class _RecipeGridPageState extends State<RecipeGridPage> {
         builder: (BuildContext context, BoxConstraints constraints) {
           final Size size = constraints.biggest;
           final double appBarHeight = size.height - statusBarHeight;
-          final String logo = appBarHeight >= 70.0 ? _kMediumLogoImage : _kSmallLogoImage;
           // Extra padding. Calculated to give about 16px on the bottom for the
           // `small` logo at its native size, and 30px for the `medium`.
           final double extraPadding = min(0.19 * appBarHeight + 5.4, 40.0);
+          final double t = (appBarHeight - kToolBarHeight) / (_kAppBarHeight - kToolBarHeight);
+          const double kOverlap = 0.28;
+          const Interval interval = const Interval(0.5 - kOverlap, 0.5 + kOverlap);
+
           return new Padding(
             padding: new EdgeInsets.only(
               top: statusBarHeight + 0.5 * extraPadding,
               bottom: extraPadding
             ),
             child: new Center(
-              child: new Image.asset(logo, fit: ImageFit.scaleDown)
```

Comment:
don't we want to only fade out the text? this will render two images on top of each other, no?